WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -122,6 +121,9 @@ - (BOOL)application:(UIApplication *)application willFinishLaunchingWithOptions:
     // Start Simperium
     [self loginSimperium];
 
+    // Listen to Local Notifications
+    [self hookLocalNotifications];
```

Comment:
I'd change the word "hook" with "observe" or "listen" which are the terms we commonly use.  I'd also consider removing the comment above as the call should make it's own purpose clear at that point.